dimagi_lake/aggregation/agg_table_schema.py

```diff
@@ -112,3 +124,70 @@ def write_to_warehouse(self):
                     drop_table(cursor, prev_table)
         finally:
             conn.close()
+
+
+
+class ChildCareMonthly(BaseTable):
+    schema = StructType(fields=[
+        StructField('domain', StringType(), True),
+        StructField('month', DateType(), True),
+        StructField('flwc_id', StringType(), True),
+        StructField('supervisor_id', StringType(), True),
+        StructField('project_id', StringType(), True),
+        StructField('district_id', StringType(), True),
+        StructField('state_id', StringType(), True),
+        StructField('case_id', StringType(), True),
+        StructField('member_case_id', StringType(), True),
+        StructField('mother_member_case_id', StringType(), True),
+        StructField('name', StringType(), True),
+        StructField('dob', DateType(), True),
+        StructField('gender', StringType(), True),
+        StructField('opened_on', DateType(), True),
+        StructField('death_date', DateType(), True),
+        StructField('age_in_months', DoubleType(), True),
+        StructField('age_in_months_end', DoubleType(), True),
+        StructField('want_nutrition_services', IntegerType(), True),
+        StructField('want_growth_tracking_services', IntegerType(), True),
+        StructField('want_counselling_services', IntegerType(), True),
+        StructField('alive_in_month', IntegerType(), True),
+        StructField('born_in_month', IntegerType(), True),
+        StructField('birth_weight', DoubleType(), True),
+        StructField('low_birth_weight', IntegerType(), True),
+        StructField('immediate_bf', IntegerType(), True),
+    ])
+
+    _aggregator = ChildCareMonthlyAggregationHelper
+    _warehouse_base_table = CHILD_CARE_MONTHLY_TABLE
+    _partition_columns = ('month',)
+
+    def aggregate(self):
+        aggregator = self._aggregator(self._database_name,
+                                      self._domain,
+                                      self._month,
+                                      self.schema)
+        agg_df = aggregator.aggregate()
+        self.write_monthly_data(agg_df)
+
+
+class ServiceEnrollment(BaseTable):
+    schema = StructType(fields=[
+        StructField('domain', StringType(), True),
+        StructField('month', DateType(), True),
+        StructField('member_case_id', StringType(), True),
+        StructField('registered_on', DateType(), True),
+        StructField('want_nutrition_services', IntegerType(), True),
+        StructField('want_growth_tracking_services', IntegerType(), True),
+        StructField('want_counselling_services', IntegerType(), True)
+    ])
+
+    _aggregator = ServiceEnrollmentAggregationHelper
+    _warehouse_base_table = SERVICE_ENROLLMENT_TABLE
+    _partition_columns = ('month',)
+
+    def aggregate(self):
```

Comment:
it looks like this is the exact same as the other monthly table. should it be extracted to a function?

Reply:
Do you mean to parent class?

Reply:
nvm. yeah. MOved it to parent class in downstream PR